.github/components.yml

```diff
@@ -54,24 +54,16 @@ NPU:
 
 HETERO:
   revalidate:
-    - CPU
     - GPU
```

Comment:
@songbell Does GPU test cases use HETERO/BATCH plugin?

Reply:
GPU + AUTO BATCH will call HETERO in some cases, but such test cases will be in BATCH/HETERO nightly.